src/assets/featureToggle.prod.json

```diff
@@ -1,4 +1,5 @@
 {
   "testToggle": false,
-  "adrToggle": false
+  "adrToggle": true,
```

Comment:
do we need to keep the adrToggle flag in this json once its all enabled? since the flags have been removed from the app i would have expected this to follow suit

Reply:
the flags haven't been removed, there was some extra floating around unused in the vehicle tech record component which I removed. I agree once it is ON in Prod we need to retire the flags completely. will add something in the backlog to do that